cmd/dm-syncer/config.go

```diff
@@ -34,6 +34,8 @@ import (
 type commonConfig struct {
 	*flag.FlagSet `json:"-"`
 
+	// task name
+	Name         string
```

Comment:
Do we need to ensure it's not empty?

Reply:
this will be checked by `subtask config`